custom/panel_templates/Default/core/reactions.tpl

```diff
@@ -43,37 +43,50 @@
-                                            <td><a href="{$reaction.edit_link}">{$reaction.name}</a></td>
-                                            <td>{$reaction.html}</td>
-                                            <td>{$reaction.type}</td>
-                                            <td>{if $reaction.enabled eq 1}
-                                                <i class="fa fa-check-circle fa-fw text-success"></i>
-                                                {else}
-                                                <i class="fa fa-times-circle fa-fw text-danger"></i>
-                                                {/if}
-                                            </td>
-                                        </tr>
-                                        {/foreach}
-                                    </tbody>
-                                </table>
-                            </div>
+                                <div class="table-responsive">
+                                    <table class="table table-borderless table-striped">
+                                        <thead>
+                                            <tr>
+                                                <td>{$NAME}</td>
+                                                <td>{$ICON}</td>
+                                                <td>{$TYPE}</td>
+                                                <td>{$ENABLED}</td>
+                                            </tr>
+                                        </thead>
+                                        <tbody id="sortable">
+                                            {foreach from=$REACTIONS_LIST item=reaction}
+                                            <tr data-id="{$reaction.id}">
+                                                <td><a href="{$reaction.edit_link}">{$reaction.name}</a></td>
+                                                <td>{$reaction.html}</td>
+                                                <td>{$reaction.type}</td>
+                                                <td>{if $reaction.enabled eq 1}
+                                                    <i class="fa fa-check-circle fa-fw text-success"></i>
+                                                    {else}
+                                                    <i class="fa fa-times-circle fa-fw text-danger"></i>
+                                                    {/if}
+                                                </td>
+                                            </tr>
+                                            {/foreach}
+                                        </tbody>
+                                    </table>
+                                </div>
                             {else}
-                            {$NO_REACTIONS}
+                                {$NO_REACTIONS}
                             {/if}
 
+                            <form action="" method="post">
```

Comment:
Now that we can display the reactions that a user has received/given, making the reaction that is actually given when a user likes a profile post configurable made sense, rather than forcing reaction ID `1`